package.json

```diff
@@ -70,7 +70,7 @@
     "lodash-es": "^4.17.15",
     "lodash.camelcase": "4.3.0",
     "minimist": "^1.2.5",
-    "ngx-bootstrap-icons": "^1.7.2",
+    "ngx-bootstrap-icons": "^1.9.1",
```

Comment:
With version 1.9 there is a peerDependency expecting angular 13, shouldn't do this if possible.